app/v1/models/response/listBenefits/CustomerIncapacityBenefit.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.models.response.listBenefits
+
+import play.api.libs.json.{Json, OFormat}
+
+case class CustomerIncapacityBenefit(submittedOn: Option[String],
```

Comment:
Similarly for this class `CustomerAddedBenefit` would probably be more accurate + no tests are present.